workflows/theiacov/wf_theiacov_illumina_pe.wdl

```diff
@@ -165,61 +160,27 @@ workflow theiacov_illumina_pe {
-            input:
-              samplename = samplename,
-              fasta = select_first([ivar_consensus.assembly_fasta]),
-              docker = organism_parameters.pangolin_docker
-          }
-        }
-        if (organism_parameters.standardized_organism == "sars-cov-2" || organism_parameters.standardized_organism == "MPXV" || defined(reference_gene_locations_bed)) {
-          # tasks specific to either sars-cov-2, MPXV, or any organism with a user-supplied reference gene locations bed file
-          call gene_coverage_task.gene_coverage {
-            input:
-              bamfile = select_first([ivar_consensus.aligned_bam, flu_track.irma_ha_bam, flu_track.irma_na_bam, ""]),
-              bedfile = select_first([reference_gene_locations_bed, organism_parameters.gene_locations_bed]),
-              samplename = samplename,
-              organism = organism_parameters.standardized_organism
-          }
-        }
-        if (organism_parameters.standardized_organism == "sars-cov-2" || organism_parameters.standardized_organism == "MPXV" || organism_parameters.standardized_organism == "rsv_a" || organism_parameters.standardized_organism == "rsv_b" || organism_parameters.standardized_organism == "WNV" || organism_parameters.standardized_organism == "flu" || organism_parameters.standardized_organism == "mumps" || organism_parameters.standardized_organism == "rubella" || organism_parameters.standardized_organism == "measles") {
-          # tasks specific to MPXV, sars-cov-2, WNV, flu, rsv_a, and rsv_b, measles, mumps, and rubella
-          call vadr_task.vadr {
-            input:
-              genome_fasta = select_first([ivar_consensus.assembly_fasta, flu_track.irma_assembly_fasta_padded]),
-              assembly_length_unambiguous = consensus_qc.number_ATCG,
-              vadr_opts = organism_parameters.vadr_opts,
-              vadr_model_file = organism_parameters.vadr_model_file,
-              max_length = organism_parameters.vadr_maxlength,
-              skip_length = organism_parameters.vadr_skiplength,
-              memory = organism_parameters.vadr_memory
-          }
-        }
       }
-      if (organism_parameters.standardized_organism == "HIV") {
-        call quasitools.quasitools as quasitools_illumina_pe {
-          input:
-            read1 = read_QC_trim.read1_clean,
-            read2 = read_QC_trim.read2_clean,
-            samplename = samplename
-        }
+      call morgana_magic.morgana_magic {
+        input:
+          samplename = samplename,
+          assembly_fasta = select_first([ivar_consensus.assembly_fasta, flu_track.irma_assembly_fasta]),
+          taxon_name = organism_parameters.standardized_organism,
+          seq_method = seq_method,
+          read1 = read_QC_trim.read1_clean,
+          read2 = read_QC_trim.read2_clean,
+          number_ATCG = consensus_qc.number_ATCG,
+          vadr_max_length = select_first([organism_parameters.vadr_maxlength, vadr_max_length]),
```

Comment:
same here -- redundant select first. was this required because of an issue during testing?